docs/developer-guide/sinks-guide.rst

```diff
@@ -0,0 +1,122 @@
+How to implement a custom Robusta sink
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What Are Robusta Sinks?
+--------------
+
+`Sinks <https://docs.robusta.dev/master/catalog/sinks/index.htm>`_
+are the destinations to which Robusta sends data. For example, when sending Robusta messages
+to Slack, Robusta uses a Slack sink.
+
+Internally, Robusta generates
+`Findings <https://docs.robusta.dev/master/developer-guide/actions/findings-api.html>`_, or messages,
+which are then formatted by each sink in an appropriate manner. For example,
+a link might be formatted one way for Slack (e.g. markdown) and in another format
+for another destination (e.g. HTML).
+
+Before reading this tutorial, make sure you are familiar with the
+`Findings API <https://docs.robusta.dev/master/developer-guide/actions/findings-api.html>`_.
+It contains explanations of the types of findings your sink will have to process.
+
+Implementation details
+--------------
+
+Sinks are located under the directory **src/robusta/core/sinks**.
```

Comment:
Can we link this and all other source code references to the relevant directory in GitHub?